airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/__init__.py

```diff
@@ -1,3 +1,19 @@
 #
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
+
+from pathlib import Path
+
+INTERNAL_POETRY_PACKAGES = [
```

Comment:
I named this `INTERNAL_POETRY_PACKAGES`  because the values of this list conditions the options value at the CLI level.
E.G `airbyte-ci test --poetry-package-path=<non-manage-by-poetry-package>` will fail because the path does not match a value in this list.

Reply:
> If we'd call it INTERNAL_PACKAGES I think the airbyte-cdk should be in there
that's fair.

>I think a nicer approach would be to glob for pyproject.toml file and keep the paths which are not in the connectors folder... Wdyt?
I don't think this will work because the CDK also has a pyproject. 

I'm fine with the name if the fact that the projects are using poetry matters, [which appears to be the case since it's required by the poetry tasks](https://github.com/airbytehq/airbyte/pull/34736/files#diff-a27b9e26d482f1d0ed938f980451495f4fd86a9ff8f88d4c654676f60279ad37R48-R49).